resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -60,1404 +60,1412 @@
+          type: basic:string
+          required: false
+          choices:
+            - label: CR
+              value: cr
+            - label: PR
+              value: pr
+            - label: SD
+              value: sd
+            - label: PD
+              value: pd
+        - name: recist_score__tenth_scan
+          label: RECIST score/10th scan
+          type: basic:string
+          required: false
+          choices:
+            - label: CR
+              value: cr
+            - label: PR
+              value: pr
+            - label: SD
+              value: sd
+            - label: PD
+              value: pd
+        - name: recist_score__description_of_recist_score
+          label: RECIST score/Description
+          type: basic:string
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__cmax
+          label: Pharmacokinetics and pharmacodynamics/Cmax
+          type: basic:string
+          description: |
+            Maximum observed serum concentration.
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__tmax
+          label: Pharmacokinetics and pharmacodynamics/Tmax
+          type: basic:string
+          description: |
+            Time of peak concentration.
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__auc
+          label: Pharmacokinetics and pharmacodynamics/AUC
+          type: basic:string
+          description: |
+            Area under curve in time of peak concentration.
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__cl
+          label: Pharmacokinetics and pharmacodynamics/CL
+          type: basic:string
+          description: |
+            Clearance.
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__t12
+          label: Pharmacokinetics and pharmacodynamics/t1_2
+          type: basic:string
+          description: |
+            Terminal elimination half-life.
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__markers
+          label: Pharmacokinetics and pharmacodynamics/Markers
+          type: basic:string
+          required: false
+        - name: pharmacokinetics_and_pharmacodynamics__description_of_pharmacokinetics_and_pharmacodynamics
+          label: Pharmacokinetics and pharmacodynamics/Description
+          type: basic:string
+          required: false
+        - name: biomarkers__her2
+          label: Biomarkers/HER2
+          type: basic:string
+          description: |
+            Human epidermal growth factor receptor 2 status.
+          required: false
+          choices:
+            - label: Positive
+              value: positive
+            - label: Negative
+              value: negative
+        - name: biomarkers__ebv
+          label: Biomarkers/EBV
+          type: basic:string
+          description: |
+            Epstein–Barr virus status.
```

Comment:
The character U+2013 "–" could be confused with the character U+002d "-", which is more common in source code.